test/fuzz-tests/participant.js

```diff
@@ -99,23 +102,48 @@ class Participant extends Actor {
     }
 
     // withdraw ETH from the rICO contract
-    async withdraw(TokenAmount = 0) {
+    async withdraw(ETH, TokenAmount) {
 
         // calculation will max out at max available ETH and tokens that can be returned.
         // TokenAmount over max available will set calculation.returned_tokens
+
         const calculation = this.helpers.utils.getAvailableEthAndTokensForWithdraw(this.helpers, this.rICO, this.address, TokenAmount);
```

Comment:
The whole point of the expected balance is to test the contract, so you can't use the contract todo the calculation for you! This calculation has to be done in JS, so that we later can compare against the current balance, once the actions is done.. Otherwise off the contract has an error we won't catch it.